client/connect.go

```diff
@@ -158,8 +159,28 @@ func (cli *Client) GetRemoteAccessToken(ctx context.Context, req RemoteAccessTok
 	return &remoteAccessToken, nil
 }
 
+// Connect establishes a connection to a remote devices
+func (cli *Client) Connect(ctx context.Context, connections map[string][]RemoteAccessTarget) error {
+	eg, ctx := errgroup.WithContext(ctx)
+
+	for deviceID, targets := range connections {
+		deviceID := deviceID
+		targets := targets
+
+		eg.Go(func() error {
+			// only print on errors, allow other connection attempts to continue
+			if err := cli.ConnectOne(ctx, deviceID, targets); err != nil {
+				fmt.Printf("error connecting to device %s: %v\n", deviceID, err)
+			}
+			return nil
+		})
+	}
+
+	return eg.Wait()
+}
+
 // Connect establishes a connection to a remote device.
-func (cli *Client) Connect(ctx context.Context, deviceID string, targets []RemoteAccessTarget) error {
+func (cli *Client) ConnectOne(ctx context.Context, deviceID string, targets []RemoteAccessTarget) error {
```

Comment:
We should try not to change existing public methods. I don't think anyone is heavily depending on it yet, but it would be a good habit to develop that we don't break public methods/APIs.

Instead, the new method should probably be called something like `ConnectMulti`.